src/sectionTitle.test.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+
+import SectionTitle, { SECTIONTITLE_CLASS } from './SectionTitle';
+
+describe('SectionTitle without action', function() {
+	let component;
+
+	beforeEach(() => {
+		component = TestUtils.renderIntoDocument(<SectionTitle />);
+	});
+	afterEach(() => {
+		component = null;
+	});
+
+	it(`check that default component has '${SECTIONTITLE_CLASS}' class`, function() {
+		const topNode = TestUtils.scryRenderedDOMComponentsWithClass(component, SECTIONTITLE_CLASS);
+
+		expect(topNode.length).toBe(1);
+	});
+
+	it('should not render a second flex item if there is no action', function() {
+		const flexItems = TestUtils.scryRenderedDOMComponentsWithClass(component, 'flex-item');
+
+		expect(flexItems.length).toBe(1);
+	});
+
+});
+
+describe('SectionTitle with action', function() {
+	const title = 'Test title';
```

Comment:
Maybe also a check to test that text and actions are in the component's render tree